server.js

```diff
@@ -12,16 +12,105 @@ mongoose.Promise = Promise;
 const port = process.env.PORT || 8080;
 const app = express();
 
+const listEndpoints = require('express-list-endpoints');
+
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// SCHEMA GOES HERE
+const { Schema } = mongoose;
+
+const ThoughtSchema = new Schema({
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140
+  },
+  heart: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date,
+    default: () => Date.now()
+  }
+});
+
+const Thought = mongoose.model('Thought', ThoughtSchema)
+
+// ROUTES GOES HERE
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.status(200).send({
+    success: true,
+    message: "OK",
+    body: {
+      content: "Welcome to Happy Thoughts API!",
+      endpoints: listEndpoints(app)
+    }
+  });
+});
+
+// GET 20 LATEST THOUGHTS
+app.get("/thoughts", async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc', _id: 'desc' }).limit(20).exec();
```

Comment:
Do you need to sort both createdAt and id in descendent order? Isn't createdAt the one that gives you the most recent post?